glean-core/ffi/src/counter.rs

```diff
@@ -15,29 +15,21 @@ define_metric!(CounterMetric => COUNTER_METRICS {
 });
 
 #[no_mangle]
-pub extern "C" fn glean_counter_test_has_value(
-    glean_handle: u64,
-    metric_id: u64,
-    storage_name: FfiStr,
-) -> u8 {
-    GLEAN.call_infallible(glean_handle, |glean| {
-        COUNTER_METRICS.call_infallible(metric_id, |metric| {
-            metric
-                .test_get_value(glean, storage_name.as_str())
-                .is_some()
-        })
+pub extern "C" fn glean_counter_test_has_value(metric_id: u64, storage_name: FfiStr) -> u8 {
+    let glean = glean_core::global_glean().lock().unwrap();
```

Comment:
Why the use of this, rather than `with_glean_value` as in `boolean.rs`?

Reply:
Oh, thanks for noticing, seems like a left-over.